### PR TITLE
Added activities & accounts endpoints

### PR DESCRIPTION
**Pull request by roiaharon-pepperi**
I've added accounts & all_activities endpoints.

### CODE REVIEW

File: papi-client.ts [hunk 1]
```diff
@@ -23,7 +24,10 @@ export class PapiClient {
     userDefinedTables = new Endpoint<UserDefinedTableRow>(this, '/user_defined_tables');
     addons = new AddonEndpoint(this);
     codeJobs = new CodeJobsEndpoint(this);
+    activities = new Endpoint<Activity>(this, '/all_activities');
```
Comment (Chasky-H) [inline papi-client.ts:27]: if you are calling all_activities, then the endpoint should be called `allActivities`,
And the object type should be (Activity | Order)

File: entities/activities.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+export interface Activity {
```
Comment (Chasky-H) [inline entities/activities.ts:1]: @roiaharon-pepperi what about TSA's,
Is there any way to add them to the entity interface? 

B/C by your code: 
``` typescript
papiClient.activities.upsert({ UUID: xxxx, TSASometsa: 'hello' })
```
will result in a compilation error.

If you can't add them to the interface, then what other options do we have?

**Comment by roiaharon-pepperi** [comment]
@Chasky-H made changes per your request

File: entities/base.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+interface NestedObjecData {
```
Comment (Chasky-H) [inline entities/base.ts:1]: typo?

**Pull request merged**